WMF Framework/WMFContentGroup+Extensions.m

```diff
@@ -242,7 +242,7 @@ - (void)setSiteURL:(nullable NSURL *)siteURL {
     self.siteURLString = siteURL.wmf_databaseKey;
 }
 
-- (void)setFullContentObject:(NSObject<NSCoding> *)fullContentObject {
+- (void)setFullContentObject:(NSObject<NSSecureCoding> *)fullContentObject {
```

Comment:
Here seems like NSArrays generally get passed in, but they could contain those Mantle objects within like WMFFeedOnThisDayEvent.

Reply:
Also a good catch, same issue as the earlier one